package.json

```diff
@@ -12,6 +12,7 @@
     "chai": "^3.4.1",
     "cucumber": "^0.9.2",
     "david": "^7.0.0",
+    "debug": "^2.2.0",
```

Comment:
These issues are caused by Circle running Node 5.1.0, and me 5.2.0 locally. 5.1.0 uses the old way of resolving dependencies and 5.2.0 the new way. Node doesn't even follow semver on its own! Do you think its a good idea to go back to Node 5.1.0 to see these errors locally and not having to debug them on CI?